src/mobility/use-vehicle.tsx

```diff
@@ -1,9 +1,11 @@
 import {useSystem} from '@atb/mobility/use-system';
 import {getRentalAppUri} from '@atb/mobility/utils';
 import {useVehicleQuery} from '@atb/mobility/queries/use-vehicle-query';
+import {useMemo} from 'react';
 
 export const useVehicle = (id: string) => {
-  const {data: vehicle, isLoading, isError} = useVehicleQuery(id);
+  const {data: vehicleData, isLoading, isError} = useVehicleQuery(id);
+  const vehicle = useMemo(() => vehicleData, [vehicleData]);
```

Comment:
Does this really do anything? 🤔

Reply:
Probably not tbh. I see this remains after debugging this [performance problem](https://github.com/AtB-AS/mittatb-app/blob/marius/select-vehicle-with-qr-scanning/src/components/map/hooks/use-auto-select-map-item.tsx#L57-L59). I'll revert it.